settings.d/dynflow.yml.example

```diff
@@ -0,0 +1,2 @@
+---
+:enabled: false
```

Comment:
Is there a reason to be false by default? One generally wants by installing the package for it to be on by default, unless there's additional configuration.

The RPM will copy the example to the real yaml.